pysmt/test/test_formula.py

```diff
@@ -610,7 +610,7 @@ def test_pickling(self):
 
         self.assertEqual(str(f), "(A & (! B))", str(f))
 
-        serialized = pickle.dumps(f)
+        serialized = pickle.dumps(f, -1)
```

Comment:
```-1``` here is confusing, you should use HIGHEST_PROTOCOL (https://docs.python.org/2/library/pickle.html#pickle.dumps)

But anyway, why is this needed?